src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -663,8 +663,8 @@ export default class AircraftModel {
         this.heading = data.heading;
         this.altitude = data.altitude;
         this.speed = data.speed;
-        this.origin = _get(data, 'origin', this.origin);
-        this.destination = _get(data, 'destination', this.destination);
+        this.origin = data.origin?.toUpperCase(),
```

Comment:
Remind me, do we need a plugin or anything for babel to make conditional chaining work or is this just native now?

Reply:
The oldest browser I've laying around is Safari 14 and the `?` operator seems to work as expected (but I can't do much more than that as Openscope itself doesn't work in Safari 14).  Seems to work in Firefox LTS and some ancient version of Chrome as well.